README.md

```diff
@@ -2,16 +2,18 @@
 
 [![Build Status](https://travis-ci.org/QueueClassic/queue_classic_admin.png)](https://travis-ci.org/QueueClassic/queue_classic_admin)
 
-An admin interface for the [queue_classic](https://github.com/ryandotsmith/queue_classic) and [queue_classic-later](https://github.com/dpiddy/queue_classic-later) gems.
+An admin interface for the [queue_classic](https://github.com/ryandotsmith/queue_classic) gem.
 
 **IMPORTANT: this branch is for QueueClassic 3. See the [queueclassic-2-support](https://github.com/dpiddy/queue_classic-later/tree/queueclassic-2-support) branch for prior version support.**
```

Comment:
Please update this with a message saying that this is 3.1 and not, and add a message saying where to find qc 3 support, like a branch named "queueclassic-3-support", I'll create it later.